### ISSUE TITLE
Sonic 2 ss 

### ISSUE THREAD

**Issue by lavagaming1** [sonicretro/s2disasm#41]
might wanna change some subroutines names in s2 ss stages and label them correctly when it comes to Objoff's thank you

**Comment by Clownacy** [comment]
Do you have any suggestions?

**Comment by lavagaming1** [comment]
yes such as changing loc_350A0 in S2 SS obj 61 boms from ss to a new name like "SSGetObCollsion" 
and maybe updating what the object manager for S2 ss is doing im gonna assume these  varables are ```	move.w	d0,objoff_30(a1)
	move.b	(a0)+,angle(a1)``` im gonna assume objoff_30 is ss_y_sub ?
